README.md

```diff
@@ -23,18 +23,15 @@ For full usage instructions, please visit the Splunk Connect for Syslog [documen
 ## Getting Support
 
 Thank you for considering SC4S for your Splunk needs.
```

Comment:
Do we want to render newlines in this section?

<img width="1048" alt="Screenshot 2023-08-09 at 17 06 22" src="https://github.com/splunk/splunk-connect-for-syslog/assets/139441697/4f897b85-1491-4f36-8e3f-ca995aee370f">

Reply:
Yes, i will correct it , thank you